src/adb/forward.js

```diff
@@ -0,0 +1,19 @@
+let debug = console.log.bind(console, '[adb/forward]');
+let exec = require('mz/child_process').exec;
```

Comment:
Yeah -- I do somewhat want to avoid having wrappers for all of the things, but definitely if our exec calls get out of hand and we're repeating our configs a ton